firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+from marionette_driver import Wait
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def tearDown(self):
+        try:
+            if self.browser.navbar.locationbar.identity_popup.is_open:
+                self.browser.navbar.locationbar.identity_popup.close(force=True)
```

Comment:
You don't need this if condition. If it's already closed the `close()` method won't do anything.